<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialWidget</class>
 <widget class="QWidget" name="MaterialWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Material Settings[*]</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>38</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Class:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="m_className">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>190</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_5">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Link:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="MaterialComboBox" name="m_linkMaterial">
     <property name="minimumSize">
      <size>
       <width>196</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Shader:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ShaderComboBox" name="m_shader">
     <property name="minimumSize">
      <size>
       <width>149</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="m_editShader">
     <property name="minimumSize">
      <size>
       <width>37</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="CheckBoxList" name="m_shaderFlags">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>262</width>
       <height>0</height>
      </size>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Sampler</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QComboBox" name="m_texUnitCombo">
     <property name="minimumSize">
      <size>
       <width>196</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QPropertyEditorWidget" name="m_texUnitProperties">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Uniform:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QComboBox" name="m_uniformCombo">
     <property name="minimumSize">
      <size>
       <width>196</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QPropertyEditorWidget" name="m_uniformProperties">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>0</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="m_saveButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>0</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MaterialComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">SceneEditorCore/MaterialComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ShaderComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">SceneEditorCore/ShaderComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>QPropertyEditorWidget</class>
   <extends>QTreeView</extends>
   <header location="global">SceneEditorCore/QPropertyEditor/QPropertyEditorWidget.h</header>
  </customwidget>
  <customwidget>
   <class>CheckBoxList</class>
   <extends>QComboBox</extends>
   <header>CheckBoxList.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
